numpy/fft/tests/test_pocketfft.py

```diff
@@ -138,6 +138,15 @@ def test_ihttf(self):
             x_herm, np.fft.ihfft(np.fft.hfft(x_herm, norm="ortho"),
                                  norm="ortho"))
 
+    def test_axes(self):
+        x = random((30, 20, 10))
+        axes = [(0, 1, 2), (0, 2, 1), (1, 0, 2), (1, 2, 0), (2, 0, 1), (2, 1, 0)]
+        for op in [np.fft.fftn, np.fft.ifftn, np.fft.rfftn, np.fft.irfftn]:
+            for a in axes:
+                rfft_tr = np.fft.rfftn(np.transpose(x, a))
+                tr_rfft = np.transpose(np.fft.rfftn(x, axes=a), a)
```

Comment:
Shouldn't this use `op`?

Reply:
Yes, I was thinking the same thing the other night.

Reply:
I fixed the oversight, and replaced inner looping over operations with parameterization.

Running `pytest --verbose numpy/fft/tests/test_pocketfft.py` shows that it seems to be working: 

```
numpy/fft/tests/test_pocketfft.py::TestFFT1D::test_ihttf PASSED                    [ 20%]
numpy/fft/tests/test_pocketfft.py::TestFFT1D::test_axes[fftn] PASSED               [ 22%]
numpy/fft/tests/test_pocketfft.py::TestFFT1D::test_axes[ifftn] PASSED              [ 23%]
numpy/fft/tests/test_pocketfft.py::TestFFT1D::test_axes[rfftn] PASSED              [ 24%]
numpy/fft/tests/test_pocketfft.py::TestFFT1D::test_axes[irfftn] PASSED             [ 25%]
numpy/fft/tests/test_pocketfft.py::TestFFT1D::test_all_1d_norm_preserving PASSED   [ 27%]
```